xml/ha_rear.xml

```diff
@@ -168,10 +168,12 @@
 
   <sect2 xml:id="sec-ha-rear-concept-versions">
    <title>&rear; version updates</title>
+
    <para>
-    &productname; &productnumber; ships with &rear; version 2.3, provided by the
-    package <package>rear23a</package>.
+    To see which versions of &rear; are available with &productname; &productnumber;,
+    run the following command:
    </para>
+<screen>&prompt.root;<command>zypper search -v rear</command></screen>
```

Comment:
Perhaps limiting the output to only ordinary RPM packages via
```
zypper search -t package -v rear
```
is better, in particular to avoid that also
source packages (.src.rpm) are shown which are
not of interest when installing software.

In general (but I don't know about what SUSE's
documentationtion style guide recommends):
Long option names could improve readability in documentation
(but could also make commands needlessly long) like
```
zypper search --type package --verbose rear
```
I was wondering because the long form `search` is used
(regardless that there is a short form `se` for it)
while in contrast the short form `-v` is used.

Reply:
You're right, I usually prefer to use the long form of commands in documentation, although I don't think it's a specific rule (which is why it's not always consistent - although in this case I just plain forgot).